include/LLVMSPIRVExtensions.inc

```diff
@@ -76,4 +76,4 @@ EXT(SPV_INTEL_task_sequence)
 EXT(SPV_INTEL_maximum_registers)
 EXT(SPV_INTEL_bindless_images)
 EXT(SPV_INTEL_2d_block_io)
-EXT(SPV_INTEL_subgroup_matrix_multiply_accumulate)
+EXT(SPV_INTEL_subgroup_matrix_multiply_accumulate)
```

Comment:
Please try to avoid removing the newline at EOF.  See also another file below.

Reply:
Thanks @svenvh 
I have resolved it in 0329e7706b406ba689ad3db6547b07ce077f5829